src/coreclr/jit/inline.cpp

```diff
@@ -499,19 +528,36 @@ void InlineContext::DumpXml(FILE* file, unsigned indent)
         mdMethodDef calleeToken  = compiler->info.compCompHnd->getMethodDefFromMethod(m_Callee);
         unsigned    calleeHash   = compiler->compMethodHash(m_Callee);
         const char* inlineReason = InlGetObservationString(m_Observation);
+        const char* name         = compiler->eeGetMethodFullName(m_Callee);
+
+        char buf[1024];
+        strncpy(buf, name, sizeof(buf));
+        buf[sizeof(buf) - 1] = 0;
+        EscapeNameForXml(buf);
 
         int offset = -1;
         if (m_Offset != BAD_IL_OFFSET)
         {
             offset = (int)jitGetILoffs(m_Offset);
         }
 
-        fprintf(file, "%*s<%s>\n", indent, "", inlineType);
-        fprintf(file, "%*s<Token>%08x</Token>\n", indent + 2, "", calleeToken);
+        fprintf(file, "%*s<Inline Success=\"%s\">\n", indent, "", m_Success ? "True" : "False");
```

Comment:
I changed `<Inline>` or `<FailedInline>` to be `<Inline Success="True/False">` because it was easier to deserialize such XML via XmlSerializer. Btw, here are the DTO objects I use: https://gist.github.com/EgorBo/828d0f5ed55826a672a07ae09bc1b5b2

Reply:
FYI, here's the deserializer code I had been using: https://github.com/AndyAyersMS/Tools/blob/main/InlineXml/analyze.cs

Reply:
Ah, I see, 
```
[XmlArrayItem("Inline", typeof(Inline))]
[XmlArrayItem("FailedInline", typeof(FailedInline))]
```
Totally forgot about it, will redo. So it won't change the format, only new fields are added.